specs/movie_reserver_spec.rb

```diff
@@ -53,15 +53,15 @@
 
   describe 'movies available' do
 
-    xit 'will list the available movies' do
+    it 'will list the available movies' do
       available_movies = reserver.available_movies(date_range)
 
       expect(available_movies.length).must_equal 11
     end
 
-    xit 'will not include rented movies' do
+    it 'will not include rented movies' do
       date_range = GreenBox::DateRange.new(Time.parse('2018-08-08'), Time.parse('2018-08-09'))
-      reserver.rent_movie('Crazy Rich Asians', date_range, 'Ada Lovelace')
+      #reserver.rent_movie('Crazy Rich Asians', date_range, 'Ada Lovelace')
```

Comment:
If you're going to comment this out, you'll need to manually add a rented movie to the system.

```ruby
reserver.rentals << Rental.new('movie title', Time.parse('2018-08-08'), Time.parse('2018-08-09'))
```